model/pod/db.go

```diff
@@ -65,3 +65,16 @@ func UpdateOne(query interface{}, update interface{}) error {
 		update,
 	)
 }
+
+// Find gets all Pods for the given query.
+func Find(query db.Q) ([]Pod, error) {
+	pods := []Pod{}
+	return pods, errors.WithStack(db.FindAllQ(Collection, query, &pods))
+}
+
+// FindByInitializing find all pods that are initialized but not running yet.
```

Comment:
FindByInitializing finds all pods that are initializing but have not started any containers.